docs/visualization/concept/basic_api_concept.md

```diff
@@ -0,0 +1,23 @@
+# Basic APIs Concept
+
+## The main basic design of the Plot class
+
+The visualization tools provided by CARET are implemented in the Plot class.
+The usage of each API of the Plot class is unified as follows:
+
+```python3
+from caret_analyze.plot import Plot
+
+... # Processing input data
+
+plot = Plot.create_[metrics]_[graph_type]_plot(data)
+plot.show()
+plot.to_dataframe()
+```
+
+The variable "plot" has two functions, show() and to_dataframe().
```

Comment:
@keita1523  
Please use inline code snippet like `Plot`.

Reply:
I modified the all function and variable in the sentence.